### PR TITLE
HTBHF-1678 Add happy path acceptance tests for the add Children's date of birth page.

### PR DESCRIPTION
**Pull request by mikeSmith08**

### CODE REVIEW

File: src/test/acceptance/features/add-children-dob.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  Background:
+    Given I have entered my details up to the add your childrens dates of birth page
+
+  Scenario: Enter one child's details
+    When I enter the details of my child who is under 3
+    And I click continue
+    Then I am shown the are you pregnant page
+
+  Scenario: Enter two children's details
+    When I enter the details of my two children who are under 3
```
Comment (dwybourn) [inline src/test/acceptance/features/add-children-dob.feature]: Should we say under 4 as that's the required age.

File: src/test/common/page/add-children-dob.js [hunk 2]
```diff
@@ -22,6 +27,71 @@ class AddChildrenDOB extends SubmittablePage {
   getPageName () {
     return 'add your childrens dates of birth'
   }
+
+  async clickAddAnotherChild () {
+    const addAnotherChildButton = await this.findById('add-another-child')
+    await addAnotherChildButton.click()
+  }
+
+  getDateLastYear () {
```
Comment (dwybourn) [inline src/test/common/page/add-children-dob.js]: The same method exists in the accessibility tests, we should move it into somewhere common.

File: src/test/acceptance/features/add-children-dob.feature (hunk 1 above)
Comment (mikeSmith08) [reply src/test/acceptance/features/add-children-dob.feature]: Yes, my mistake, I had the phrase 3 or younger in my head and just did this wrongly.

File: src/test/common/page/add-children-dob.js (hunk 2 above)
Comment (mikeSmith08) [reply src/test/common/page/add-children-dob.js]: Yes, I've been thinking about doing some refactoring of date based tests so I'll look at it in another PR.

File: src/test/acceptance/features/navigation.feature [hunk 3]
```diff
@@ -17,6 +17,7 @@ Feature: Application process navigation is controlled
       | are you pregnant                      |
       | card address                          |
       | do you have children three or younger |
+      | add your childrens dates of birth     |
```
Comment (gethyn1) [inline src/test/acceptance/features/navigation.feature]: Should be `enter your childrens dates of birth` to match other tests

File: src/test/common/steps/navigation.js [hunk 4]
```diff
@@ -28,6 +29,7 @@ const EMAIL_ADDRESS_PAGE = 'email address'
 const SEND_CODE_PAGE = 'send code'
 const ENTER_CODE_PAGE = 'enter code'
 const CHECK_PAGE = 'check details'
+const ADD_CHILDREN_DOB_PAGE = 'add your childrens dates of birth'
```
Comment (gethyn1) [inline src/test/common/steps/navigation.js]: Same as above comment `ADD_CHILDREN_DOB_PAGE` -> `ENTER_CHILDREN_DOB_PAGE`

File: src/test/acceptance/features/navigation.feature (hunk 3 above)
Comment (mikeSmith08) [reply src/test/acceptance/features/navigation.feature]: Yes, this will be done in another PR.

File: src/test/common/steps/navigation.js (hunk 4 above)
Comment (mikeSmith08) [reply src/test/common/steps/navigation.js]: Yes, this will be done in another PR, there's a separate sub-task for it now.

**Pull request merged**